test/www/dir/meson.build

```diff
@@ -0,0 +1,7 @@
+# SPDX-FileCopyrightText: 2021 Andrea Pappacoda
```

Comment:
Could you remove this file from 'test/www/dir' directory?

Reply:
Unfortunately I can't.
The "old" tests are run in-tree, and require `ca-bundle.crt`, `image.jpg` `www/dir*/*` to be ran. 
In-tree builds are disallowed in Meson (and discouraged in CMake), so it is necessary to copy the required files in the build directory to recreate the same directory structure as the one found in the source tree (that is, `test/ca-bundle.crt` → `build/test/ca-bundle.crt`, etc). Copying files to the build dir is accomplished by the `configure_file(input: 'filename', output: 'filename', copy: true)` function, where the input is a file name relative to the same directory as the current `meson.build` file. Copying a file from a subdirectory is not allowed, as Meson guaranties that a file in a certain file in the build dir (like `build/test/www/dir/index.html`) has to come from the same path of the source tree (`test/www/dir/index.html`).
This makes debugging easier, but obviously has the downside of having to create a `meson.build` file in the same dir as the files you want to copy. So I can't remove these three files :/

Reply:
'www', 'www2' and 'www3' are just htdocs folders, and I may add more such folders like 'www4' in the future. I would not like to remember adding 'meson.build' file to each new data directory, and it puts an extra burden on me. I personally don't care for testing on Meson build system, because I only use just plain 'Make' files. (I also don't use CMake file in this project personally, and I think it is not used for testing as well.) Thanks for your understanding!

Reply:
I understand your concern, but don't worry, the worst that could happen if you don't add the meson.build file is a failed test, better than no tests at all :)

I've committed to maintain the Meson part of this repo, so if something Meson-related has issues I'll fix it as soon as possible (the same way @sum01 is the "maintainer" of the CMake portion of cpp-httplib).

If for whatever reason I stop maintaining something that needs to be updated (like the tests) you're always free to remove it (like you did with CMake some time ago).

Having tests that ensure that Meson users do not use a broken build of the library (when not using the header-only version) are important to ensure that cpp-httplib will always work fine for everyone.

Reply:
Glad to see your willingness! I'll merge it.